pkg/ingester/ingester_v2.go

```diff
@@ -106,6 +106,9 @@ type userTSDB struct {
 	seriesInMetric *metricCounter
 	limiter        *Limiter
 
+	globalSeriesCount *atomic.Int64 // Shared across all userTSDB instances created by ingester.
```

Comment:
What happens when we close TSDB (eg. because idle)? Is this updated correctly? Can we update existing unit tests to check it too, please?

Reply:
Normally yes (head is empty already), except for case when we're closing TSDB because of discovered deletion mark. Updated unit tests and fixed it.